tests/lsp_features/folding_range.zig

```diff
@@ -12,6 +12,15 @@ test "foldingRange - empty" {
     try testFoldingRange("", &.{});
 }
 
+test "foldingRange - container type without members" {
+    try testFoldingRange(
+        \\const S = struct {
+        \\};
+    , &.{
+        .{ .startLine = 0, .startCharacter = 16, .endLine = 1, .endCharacter = 0 },
```

Comment:
the folding range should start after '{'
```suggestion
        .{ .startLine = 0, .startCharacter = 18, .endLine = 1, .endCharacter = 0 },
```